README.md

```diff
@@ -236,6 +253,57 @@ open-attestation document-store revoke --network ropsten --address 0x19f89607b52
 ✔  success   Document/Document Batch with hash 0x0c1a666aa55d17d26412bb57fbed96f40ec5a08e2f995a108faf45429ae3511f has been revoked on 0x19f89607b52268D0A19543e48F790c65750869c6
 ```
 
+## Paymaster
+
+### set-target
+
+Registers a contract with a paymaster deployed on the blockchain is willing to pay for
+
+```bash
+open-attestation paymaster set-target --targetAddress <GSN_CAPABLE_DOCUMENT_STORE_ADDRESS> --paymasterAddress <PAYMASTER_ADDRESS> [options]
+```
+
+Example - with private key set in `OA_PRIVATE_KEY` environment variable (recommended). [More options](#providing-the-wallet).
+
+```bash
+open-attestation paymaster set-target --network ropsten --targetAddress 0x9Eb76E132fCA96779A5225419352Fb1B3B5Fd706 --paymasterAddress 0xcB94584760bCA09e9fa7117C4eE966814f17a306
+
+✔  success   Contract with address 0x9Eb76E132fCA96779A5225419352Fb1B3B5Fd706 has been registered on paymaster 0xcB94584760bCA09e9fa7117C4eE966814f17a306
+```
+
+### remove-target
```

Comment:
on naming
- set-target makes me think you can only add one contract
- remove-target makes me think you can add multiple contracts

I would suggest to rename set-target to add-target